### PR TITLE
Improved the debug messaging so it now goes to the debugger

### PR DESCRIPTION
**Pull request by nir9**
Debug messages now go to the debugger/dbgview instead of going to nowhere (there is no console attached so stderr doesn't go anywhere)

### CODE REVIEW

File: dwm-win32.c [hunk 1]
```diff
@@ -465,10 +456,44 @@ drawtext(const wchar_t *text, unsigned long col[ColLast], bool invert) {
...
+    size_t buffer_num_of_chars;
+    wchar_t program_name[] = L"dwm-win32: ";
 
     va_start(ap, errstr);
-    vfwprintf(stderr, errstr, ap);
-    fflush(stderr);
+
+    num_of_chars = _vscwprintf(errstr, ap);
+    if (num_of_chars == -1)
+    {
```
Comment (prabirshrestha) [inline dwm-win32.c:468]: need to move `{` to previous line

File: dwm-win32.c [hunk 2]
```diff
@@ -465,10 +456,44 @@ drawtext(const wchar_t *text, unsigned long col[ColLast], bool invert) {
...
+    buffer = (wchar_t*)calloc(buffer_num_of_chars, sizeof(wchar_t));
+    if (buffer == NULL)
+    {
+        OutputDebugStringW(L"calloc failed in eprint");
+        return;
+    }
+
+    if (wcscpy_s(buffer, buffer_num_of_chars, program_name) != 0)
+    {
+        OutputDebugStringW(L"wcscpy_s failed in eprint");
```
Comment (prabirshrestha) [inline dwm-win32.c:483]: need to free buffer here

File: dwm-win32.c [hunk 3]
```diff
@@ -465,10 +456,44 @@ drawtext(const wchar_t *text, unsigned long col[ColLast], bool invert) {
...
+        return;
+    }
+
+    if (vswprintf(buffer + wcslen(program_name), num_of_chars + 1, errstr, ap) < 0)
+    {
+        OutputDebugStringW(L"vswprintf failed in eprint");
```
Comment (prabirshrestha) [inline dwm-win32.c:489]: need to free buffer here

**Comment by nir9** [comment]
Fixed review comments

**Comment by prabirshrestha** [comment]
Merged. Thanks.

**Pull request merged**